src/frontend/App.js

```diff
@@ -89,6 +93,29 @@ class App {
 		return tempDiv.innerHTML;
 	}
 
+	addTargetBlack( message ) {
```

Comment:
Is here type in the name?  Like `Block` instead of `Black`?

Reply:
Yea. It's supposed to be Blank. Thanks.